pom.xml

```diff
@@ -216,6 +216,8 @@
     <pitest.plugin.timeout.factor>10</pitest.plugin.timeout.factor>
     <pitest.plugin.timeout.constant>50000</pitest.plugin.timeout.constant>
     <pitest.plugin.threads>4</pitest.plugin.threads>
+    <sonar.language>java</sonar.language>
+    <sonar.profile>checksyle-profile</sonar.profile>
```

Comment:
is it possible to define such variables in plugin config ?
the less scope of variables the better.